app/javascript/lib/metrics-api.js

```diff
@@ -78,8 +77,12 @@ export function logClick(event) {
 /**
  * Log click on link, i.e. anchor (<a ...) tag
  */
-function logClickLink(target) {
-  const props = { text: target.text }
+export function logClickLink(target) {
+  const props = {
+    text: target.text,
+    classList: 'classList' in target? [...target.classList] : '',
```

Comment:
Blocking: to keep the logged type the same and avoid possible future Mixpanel validation issues, please use `[]` instead of `''` for empty `classList`.

Non-blocking: `Array.from` is more readable than spread.

```suggestion
    classList: 'classList' in target? Array.from(target.classList) : [],
```